token/twoxtx-setup.sh

```diff
@@ -25,4 +25,11 @@ if [[ ! -f twoxtx-solana/.twoxtx-patched ]]; then
 fi
 
 ../patch.crates-io.sh twoxtx-solana
+repo="token/twoxtx-solana"
+if sed -n '/exclude = \[/,/\]/p' ../Cargo.toml | grep -q "$repo"; then
+  echo "$repo is already excluded"
+else
+  sed -i'' ../Cargo.toml -e "/exclude/a \ \ \"$repo/\","
```

Comment:
nitty nit: is the slash required? I don't see it used for any of the other workspace items
```suggestion
  sed -i'' ../Cargo.toml -e "/exclude/a \ \ \"$repo\","
```

Reply:
Nope, looks like it's not necessary. I wonder if all the other lines could be condensed to just `farms`...